tests/gobcore/workflow/test_start_commands.py

```diff
@@ -191,3 +192,163 @@ def test_get_all(self, mock_load, mock_open, mock_start_command):
         start_commands.commands = MagicMock()
 
         self.assertEqual(start_commands.commands, start_commands.get_all())
+
+
+class Struct:
```

Comment:
Poor choice of type name? This is not a struct.
I think that argparse calls it a namespace.
https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.parse_args
https://docs.python.org/3.10/library/argparse.html#argparse.Namespace